### DISCUSSION TITLE
After installing python 3.10 in pyenv, I am receiving import errors on all my virtual envs

### DISCUSSION THREAD

**Discussion by mattmitchell22** [pyenv/pyenv#2166] [Q&A] [answered]
I installed python 3.10 yesterday through pyenv and created a new virtual environment with that version of python. Everything worked fine. Now today, every virtual environment I have is giving me import errors when attempting to run python manage.py runserver for Django. Any ideas?

**Comment by native-api** [comment, marked as answer]
From your description, it seems that the Django installation that the `manage.py` belongs to belongs to a different, earlier env than the one you mentioned here.

If that's so, you need make sure the right env [is selected](https://github.com/pyenv/pyenv#choosing-the-python-version) when you work with the `manage.py`.

**Reply by mattmitchell22** [reply]
It's some pathing issue I can't figure out. I have continued fooling with it, and now when I try to run the local django server in a virtualenv it says it can't find any of the packages installed. This is the case for every virutalenv I have (they are all for django projects).

**Reply by mattmitchell22** [reply]
I am not sure if this is normal, but when I run pyenv versions, it gives a named version of the virutal env, and also a version of the virtual env under 3.9.0


![image](https://user-images.githubusercontent.com/34243216/142774751-8f5a1390-22a7-40e4-9db8-543c84b8cc3b.png)

**Reply by native-api** [reply]
> I am not sure if this is normal, but when I run pyenv versions, it gives a named version of the virutal env, and also a version of the virtual env under 3.9.0

This is normal.

**Reply by native-api** [reply]
> It's some pathing issue I can't figure out. I have continued fooling with it, and now when I try to run the local django server in a virtualenv it says it can't find any of the packages installed. This is the case for every virutalenv I have (they are all for django projects).

As I said, find out which of the environments have your packages and make sure it is selected when you need it.

**Reply by mattmitchell22** [reply]
Something is off with pathing somewhere. Since this issue came up I tried installing django another env I just created, and it did not install django in the env, but to my system python.

**Reply by native-api** [reply]
I cannot say anything else with the information given. If you create an issue and provide the required diagnostic information, I'll be able to say more.

**Reply by mattmitchell22** [reply]
What diagnotic information are you looking for?

**Comment by mattmitchell22** [comment]
> As I said, find out which of the environments have your packages and make sure it is selected when you need it.

I am not sure what you mean by this. Every environment is self-contained with the packages they need.

**Reply by native-api** [reply]
Check the README on this.
